### DISCUSSION TITLE
Idea: E-Bicycle ETA Options 🚲 🔌 🕐

### DISCUSSION THREAD

**Discussion by colincowie** [organicmaps/organicmaps#5464] [Ideas]
I love using Organic Maps for riding my Electric Bicycle! Like most mapping software the "cycling" ETA assuming you are riding a regular bike and not a pedal assisted E-Bike which is a bit faster on average.  

**It would be awesome if there was an option to adjust the cycling ETA calculation for E-Bike riders!** 

In the US most E-Bikes support up 33 km/h and many go up to 40 km/h assisted 

This could be implemented with some sort of preference setting for e-bike vs normal bike or alternatively a preference setting for how fast you normally cycle.

Please let me know if can help clarify this feature idea. Thank you the great app! 😄

**Comment by biodranik** [comment]
What is the main benefit/goal of this feature? Is it only about more accurate ETAs? Or also about something else? Can routes be different for a normal bike vs e-bike? For example, we can remove "climbing up" penalty for assisted bikes, and then a different route can be built.

Let's collect all UX and route-related differences here, maybe together with examples from other apps. Then a router may be configured with an "I have an e-bike" setting for better routes.

**Reply by ghost** [reply]
Every cyclist has a different cruise speed. On one of my bikes I tend to go about 15km/h, and with the other I tend to go about 20 or 25km/h. When I use brouter.de to plan my route it makes a big difference when accurately planning. And that speed isn't dependant on road type. 
I would love to be able to have similar customisations for bike routing with organic maps. Not as many as brouter.de, perhaps, but at least the speed.
